src/ucp/core/ucp_mm.c

```diff
@@ -431,6 +431,7 @@ ucp_memh_get_slow(ucp_context_h context, void *address, size_t length,
                                        uct_flags);
     if (status == UCS_OK) {
         *memh_p = memh;
+        ucs_trace("memh %p: get slow", *memh_p);
```

Comment:
1. maybe add details such as address, length, md_map?
2. swap with line 433
3. maybe smth like  `ucs_trace("memh %p: registered address %p length %zu md_map %"PRIx64, ..., *memh_p);`

Reply:
ok, initially I added only what was really useful in my case, I was seeking a memory leak.